tests/transformers/electra/test_modeling.py

```diff
@@ -77,6 +78,13 @@ def prepare_config_and_inputs(self):
             token_type_ids = ids_tensor([self.batch_size, self.seq_length],
                                         self.type_vocab_size)
 
+        inputs_embeds = None
+        if self.use_inputs_embeds:
+            inputs_embeds = floats_tensor(
+                [self.batch_size, self.seq_length, self.embedding_size])
+            # In order to use inputs_embeds, input_ids needs to set to None
+            input_ids = None
```

Comment:
if not `use_inputs_embeds`, it should not prepare the `input_ids` tensor in `prepare_config_and_inputs` method.